src/sv-pipeline/scripts/downstream_analysis_and_filtering/plot_svcount_outliers.R

```diff
@@ -0,0 +1,89 @@
+getJitter <- function(x,dens){
+  jit <- dens$y[head(which(dens$x>=x),1)]
+  return(jit)
+}
+jitterX <- function(y,dens){
+  return(jitter(1,amount=getJitter(y,dens)))
+}
+
+
+library(optparse)
+###List of command-line options
+option_list <- list(
+  make_option(c("-i","--input"), type="character", 
+              help="stat table with count of SVs per sample"),
+  make_option(c("-s","--svcount"), type="character", 
+              help="stat table with count of overall SVs sites"),
+  make_option(c("-o","--outputpath"), type="character", 
+              help="output folder for all plots"),  
+  make_option(c("-m","--max"), type="integer",
+              help="max number of IQR to plot")
+ )
+
+
+###Get command-line arguments & options
+args <- parse_args(OptionParser(usage="%prog INFILE OUTDIR",
+                                option_list=option_list),
+                   positional_arguments=TRUE)
+
+opts <- args$options
+dat=read.table(opts$input, header = T)
+all_counts=read.table(opts$svcount)
+all_counts=all_counts[all_counts[,1]>0,]
+output_prefix = gsub('.txt','',opts$input)
+
+svtype=unique(dat$svtype)
+x_max= opts$max
+for(i in svtype){
```

Comment:
`i` -> `svtype`

Reply:
that'll be a bug and overwrite the original `svtype`

Reply:
Sorry, I guess I meant something like, rename `svtype` to `svtypes` and then rename `i` to `svtype`. But again, not really a big deal, just trying to improve readability.